build.assets/tooling/cmd/changelog/README.md

```diff
@@ -0,0 +1,81 @@
+# changelog
+
+This script generates a changelog for a release.
+
+## Usage
+
+```
+usage: changelog [<flags>]
+
+Flags:
+  --[no-]help              Show context-sensitive help (also try --help-long and --help-man).
+  --baseBranch=BASEBRANCH  The base release branch to generate the changelog for. It will be of the form branch/v* ($BASE_BRANCH)
+  --baseTag=BASETAG        The tag/version to generate the changelog from. It will be of the form vXX.Y.Z, e.g. v15.1.1 ($BASE_TAG)
```

Comment:
```suggestion
  --base-tag=BASETAG        The tag/version to generate the changelog from. It will be of the form vX.Y.Z, e.g. v15.1.1 ($BASE_TAG)
```

Feels more natural than camelCase flags.